run-tests.php

```diff
@@ -3221,8 +3256,13 @@ function parse_conflicts(string $text): array
     return array_map('trim', explode("\n", trim($text)));
 }
 
-function show_result($result, $tested, $tested_file, $extra = '', $temp_filenames = null)
-{
+function show_result(
+    string $result,
+    string $tested,
+    string $tested_file,
+    string $extra = '',
+    ?string $temp_filenames = null
```

Comment:
```suggestion
    ?array $temp_filenames = null
```
Also, this parameter is unused and can be dropped. But maybe better as a separate patch.

Reply:
I'll do that as a separate PR with doing some other clean-up hopefully.